pkg/security/tests/network_device_test.go

```diff
@@ -133,3 +138,157 @@ func TestNetDevice(t *testing.T) {
 		}
 	})
 }
+
+func TestTCFilters(t *testing.T) {
+	checkKernelCompatibility(t, "RHEL, SLES and Oracle kernels", func(kv *kernel.Version) bool {
+		// TODO: Oracle because we are missing offsets
+		return kv.IsRH7Kernel() || kv.IsOracleUEKKernel() || kv.IsSLESKernel()
+	})
+
+	// skip the test to avoid nested namespaces issues
+	if testEnvironment == DockerEnvironment {
+		t.Skip("skipping tc filters test in docker")
```

Comment:
We def disable this feature in the non-test path right?

Reply:
for posterity, discussed offline. Only this test doesn't work on docker, not the feature. But there's no value-add with having this test work on both host and docker, so requested just a comment clarifying this.